OverflowSet: Don't assume role or orientation. (#11658)
OfficeDev/office-ui-fabric-react: packages/office-ui-fabric-react/src/components/OverflowSet/docs/OverflowSetOverview.md

### Accessibility

> By default, the OverflowSet is simply `role=group`. If you used as a menu, you will need to add `role="menubar"` and add proper aria roles to each rendered item (`menuitem`, `menuitemcheckbox`, `menuitemradio`)

Reviewer: ```suggestion
By default, the OverflowSet is simply `role=group`. If used as a menu, then `role="menubar"` should be added, as well as the proper aria roles for each rendered item (`menuitem`, `menuitemcheckbox`, `menuitemradio`).
```
